test/project-service.ts

```diff
@@ -273,6 +292,153 @@ describe("Project Service Tests", () => {
 			projectIntegrationTest.createProject(projectName).wait();
 			projectIntegrationTest.assertProject(tempFolder, projectName, options.appid).wait();
 		});
+
+		describe("project name validation tests", () => {
+			let validProjectName = "valid";
+			let invalidProjectName = "1invalid";
+
+			beforeEach(() => {
+				hasPromptedForString = false;
+			})
```

Comment:
missing `;`
In fact this is my mistake, linter should catch such errors, but somehow I've excluded tests from linting :crying_cat_face: